ckan/templates-midnight-blue/package/base_form_page.html

```diff
@@ -15,22 +34,79 @@
 
 {% block secondary_content %}
   {% block info_module %}
-    <section class="module module-narrow module-shallow">
```

Comment:
The `module module-narrow module-shallow` classes provided the padding/margins for mobile, we should add this back, as there is no longer any padding on mobile.